mailbox/event/event-rabbitmq/src/main/java/org/apache/james/mailbox/events/GroupRegistration.java

```diff
@@ -125,11 +125,12 @@ GroupRegistration start() {
             .then();
     }
 
-    private Flux<Void> consumeWorkQueue() {
-         return receiver.consumeManualAck(queueName.asString(), new ConsumeOptions().qos(EventBus.EXECUTION_RATE))
+    private Disposable consumeWorkQueue() {
```

Comment:
What are you doing with this Disposable?

Reply:
It is stored in `receiverSubscriber`, as in the first version.

The issue with your version is that went from a `Disposable` on a `Flux<Delivery>` and `deliver` (originated in `receiver.consumeManualAck`), to a `Disposable` on a `GroupWorkQueue` creation.